appgate/syncer/git.py

```diff
@@ -699,18 +689,27 @@ async def _create(self, e: Entity_T, register_commit: bool = True) -> EntityClie
         p.mkdir(exist_ok=True)
         file = git_dump(e, self.api_spec, p, self.resolution_conflicts)
         if register_commit:
-            self.commits.append((file, "ADD"))
+            self.commits.append(
+                (p, GitCommitState(entity=e, path=file, operation="ADD"))
```

Comment:
we were returning `file` now we return `p`. Is that on purpose?

Reply:
Unintentional. Reverting back to `file`